src/webgl/shaders/cubeFragment.frag

```diff
@@ -0,0 +1,22 @@
+// Adapted from: https://learnopengl.com/PBR/IBL/Diffuse-irradiance
+IN vec3 localPos;
```

Comment:
In my opinion I feel for maximum compatibility, we can do this one in GL ES 100 instead of 300? Using attribute instead of IN, using the spacial variable gl_FragColor instead of defining an OUT_COLOR, and using texture2D() instead of texture()? What you feel?

Reply:
Well I started from that only , and was getting a lot of errors and from Dave's suggestion of using the `macros` instead resolved the errors , I again tried doing what u have suggested today, but strangely that gives in more errors! So I think we should stick to using these macros only :-)
You can refer to Dave's suggestion here in this comment:
https://github.com/processing/p5.js/pull/6665#discussion_r1437846579

Reply:
Hmmm...It looks like you need to use both webgl-1 and webgl-2 modes on shaders.